.gitignore

```diff
@@ -20,3 +20,10 @@
 /log/*
 !/log/.keep
 /tmp
+
+# Ignore all Node modules
+/node_modules
+
+# Ignore all the config files
```

Comment:
I'm really bad at gardening so in this way I prevent myself from committing the wrong things.